examples/Getting_Started/02_Geometry_Optimization/httpx/run.py

```diff
@@ -18,6 +18,8 @@
 foldername = "output"
 base_url = os.getenv("PM_API_BASE_URL", "https://api.promethium.qcware.com")
 gpu_type = os.getenv("PM_GPU_TYPE", "a100")
+workflow_timeout = int(os.getenv("PM_WORKFLOW_TIMEOUT", 864000))
+task_timeout = int(os.getenv("PM_TASK_TIMEOUT", 864000))
```

Comment:
Custom timeouts are not something we need to include on every example, let's just show them in the context of one or two of them.